packages/relayer/processor/process_message.go

```diff
@@ -243,11 +245,11 @@ func (p *Processor) sendProcessMessageCall(
 		auth.GasLimit = 3000000
 	} else {
 		// otherwise we can estimate gas
-		gas, cost, err = p.estimateGas(ctx, event.Message, proof)
+		gas, _, err = p.estimateGas(ctx, event.Message, proof)
```

Comment:
instead of `_`, lets just remove `cost` from being returned by this function now.